meta-mel/conf/local.conf.append.cyclone5

```diff
@@ -60,3 +60,17 @@ MPV = "${@bb.utils.contains('INCLUDE_MPV', 'yes', 'mpv', '', d)}"
 LIC_WHITELIST_MPV = "${@bb.utils.contains('INCLUDE_MPV', 'yes', 'commercial_ffmpeg commercial_x264', '', d)}"
 LICENSE_FLAGS_WHITELIST += "${LIC_WHITELIST_MPV}"
 CORE_IMAGE_EXTRA_INSTALL_append += "${MPV}"
+
+# Certain multimedia formats requires packages that have IP issues and other
+# restrictions and hence they are not included in the root file-system by default.
+# If you agree with their licenses and want to include those license restricted
+# packages in your rootfs then set INCLUDE_COMMERCIAL_MULTIMEDIA to yes.
+
+INCLUDE_COMMERCIAL_MULTIMEDIA = "no"
+
+COMMERCIAL_MULTIMEDIA = "${@bb.utils.contains('INCLUDE_COMMERCIAL_MULTIMEDIA', 'yes', 'gstreamer1.0-plugins-ugly gstreamer1.0-libav', '', d)}"
+LIC_WHITELIST_COMMERCIAL = "${@bb.utils.contains('INCLUDE_COMMERCIAL_MULTIMEDIA', 'yes', ' commercial_gstreamer1.0-plugins-ugly \
+                                            commercial_gstreamer1.0-libav commercial_lame commercial_libmad commercial_mpeg2dec ', '', d)}"
```

Comment:
contains() is for whitespace-separated lists, not for checking if the value is equal to something. also, we have an existing function to handle boolean-style variables. I also think the lists of elements should be separated from the conditional logic to make it easier to see at a glance what's being included. Also a local.conf.append is at the end of the file, so there's no need to use _append on CORE_IMAGE_EXTRA_INSTALL. I.e.:

```bitbake
INCLUDE_COMMERCIAL_MULTIMEDIA = "no"

COMMERCIAL_MEDIA_PACKAGES = "gstreamer1.0-plugins-ugly gstreamer1.0-libav"
COMMERCIAL_LIC_FLAGS = "commercial_gstreamer1.0-plugins-ugly commercial_gstreamer1.0-libav commercial_lame commercial_libmad commercial_mpeg2dec"

LICENSE_FLAGS_WHITELIST .= "${@' ${COMMERCIAL_LIC_FLAGS}' if bb.utils.to_boolean('${INCLUDE_COMMERCIAL_MULTIMEDIA}') else ''}"
CORE_IMAGE_EXTRA_INSTALL .= "${@' ${COMMERCIAL_MEDIA_PACKAGES}' if bb.utils.to_boolean('${INCLUDE_COMMERCIAL_MULTIMEDIA}') else ''}"
```